<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-EVAHJRN" ignored="0" name="Default test" tests="1" failures="0" timestamp="18 Apr 2018 08:56:50 GMT" time="0.203" errors="0">
  <testcase name="@BeforeTest startService" time="0.156" classname="com.fluke.screens.rough.SampleTest">
    <failure type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException" message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\DELL\AppData\Local\Programs\appium-desktop\resources\app\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1, --log, C:\Users\DELL\eclipse-workspace\AutomationFramework\Appium.log] 
">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\DELL\AppData\Local\Programs\appium-desktop\resources\app\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1, --log, C:\Users\DELL\eclipse-workspace\AutomationFramework\Appium.log] 

	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:149)
	at com.fluke.screens.rough.SampleTest.startService(SampleTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NoSuchMethodError: com.google.common.util.concurrent.SimpleTimeLimiter.create(Ljava/util/concurrent/ExecutorService;)Lcom/google/common/util/concurrent/SimpleTimeLimiter;
	at org.openqa.selenium.net.UrlChecker.<init>(UrlChecker.java:62)
	at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:113)
	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:136)
	... 24 more
]]>
    </failure>
  </testcase> <!-- @BeforeTest startService -->
  <testcase name="@AfterTest stopService" time="0.0" classname="com.fluke.screens.rough.SampleTest">
    <skipped/>
  </testcase> <!-- @AfterTest stopService -->
  <testcase name="test" time="0.0" classname="com.fluke.screens.rough.SampleTest">
    <skipped/>
  </testcase> <!-- test -->
</testsuite> <!-- Default test -->
